File: dart-jaguar/lib/model/main_get_api_keys_resp.dart
import 'package:jaguar_serializer/jaguar_serializer.dart';

part 'main_get_api_keys_resp.jser.dart';

class MainGetApiKeysResp {
  
  final String expiry;
  
  final String label;
  
  final String token;
  
  final String tokenHash;
  

  MainGetApiKeysResp(
    

{
     this.expiry = null,  
     this.label = null,  
     this.token = null,  
     this.tokenHash = null 
    
    }
  );

  @override
  String toString() {
    return 'MainGetApiKeysResp[expiry=$expiry, label=$label, token=$token, tokenHash=$tokenHash, ]';
  }
}

@GenSerializer()
class MainGetApiKeysRespSerializer extends Serializer<MainGetApiKeysResp> with _$MainGetApiKeysRespSerializer {

}

File: scala-akka-http-server/src/main/scala/io/swagger/server/model/Main.getApiKeysResp.scala
package io.swagger.server.model


/**
 * @param expiry 
 * @param label 
 * @param token 
 * @param tokenHash 
 */
case class Main.getApiKeysResp (
  expiry: Option[String],
  label: Option[String],
  token: Option[String],
  tokenHash: Option[String]
)

